webapp/app/js/directives/clonejobs.js

```diff
@@ -236,7 +236,10 @@ treeherder.directive('thCloneJobs', [
             jobStatus.key = key;
             if(parseInt(job.failure_classification_id, 10) > 1){
                 jobStatus.value = job.job_type_symbol + '*';
-            }else{
+                if (jobStatus.btnClassClassified) {
+                    jobStatus.btnClass = jobStatus.btnClassClassified;
```

Comment:
show the "classified" version of a failed jobs with a transparent background when it's been, well... classified.